src/ValueObject/ProcessResult.php

```diff
@@ -33,8 +33,12 @@ public function getSystemErrors(): array
     /**
      * @return FileDiff[]
      */
-    public function getFileDiffs(): array
+    public function getFileDiffs(bool $onlyWithChanges = true): array
```

Comment:
In most cases we will just be interested in the file diffs where there was an actual change, so we filter out those where there was no change recorded